qt/TorrentDelegate.cc

```diff
@@ -578,6 +584,38 @@ void TorrentDelegate::drawTorrent(QPainter* painter, QStyleOptionViewItem const&
 
     painter->setFont(layout.name_font);
     painter->drawText(layout.name_rect, Qt::AlignLeft | Qt::AlignVCenter, layout.nameText());
+    painter->setFont(layout.labels_font);
+
+    // labels
+    int width_used = 0;
+    for (auto label_text : tor.labels())
+    {
+        QRect label_rect = QFontMetrics{ layout.labels_font }.boundingRect(label_text);
+        label_rect.moveTopLeft(layout.labels_rect.topLeft() + QPoint{ width_used, 0 });
+        width_used += label_rect.width();
+
+        // gap between labels
+        width_used += 5;
+
+        QPalette::ColorGroup cg = is_item_enabled ? QPalette::Normal : QPalette::Disabled;
+        QPalette::ColorRole cr = (option.features & QStyleOptionViewItem::ViewItemFeature::Alternate) ? QPalette::Base :
+                                                                                                        QPalette::AlternateBase;
+
+        if (cg == QPalette::Normal && !is_item_active)
+        {
+            cg = QPalette::Inactive;
+        }
+
+        if (is_item_selected)
+        {
+            cr = QPalette::Highlight;
+        }
+
+        painter->setBrush(option.palette.brush(cg, cr));
```

Comment:
@UI_Revierers: All these color changes are not really necessary, but I think the label boxes looked kind of bland without them.